Src/BootCamp.Chapter/MsFactory.cs

```diff
@@ -7,9 +7,28 @@ namespace BootCamp.Chapter
 {
     public class MsFactory
     {
+        private DesktopComputer _msComputer;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make '_msComputer' 'readonly'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5121609)